### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: pytest_timeout.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def cancel():
     elif params.method == 'thread':
         timer = threading.Timer(params.timeout, timeout_timer,
                                 (item, params.timeout))
-        timer.name = "%s %s" (__name__, item.nodeid)
+        timer.name = "%s %s" % (__name__, item.nodeid)
```
Comment (nicoddemus) [inline pytest_timeout.py]: Heh

**Comment by nicoddemus** [comment]
Merging so we can get CI in place right away.

File: pytest_timeout.py (hunk 1 above)
Comment (graingert) [reply pytest_timeout.py]: :(

Comment (nicoddemus) [reply pytest_timeout.py]: I didn't notice it too. 😆

**Pull request merged**